docs/features/software-catalog/descriptor-format.md

```diff
@@ -444,6 +446,22 @@ Apart from being a string, the software catalog leaves the format of this field
 open to implementers to choose. Most commonly, it is set to the ID or email of a
 group of people in an organizational structure.
 
+### `spec.system` [optional]
+
+Links the system that the component belongs to, e.g. `artist-engagement-portal`.
```

Comment:
same here, the word "system" could be made a link to the heading for System within this page

Reply:
Yeah, we could improve hot linking in the whole document, I will take a look